lib/Dialect/FIRRTL/FIRRTLFolds.cpp

```diff
@@ -50,17 +50,14 @@ static IntegerAttr getIntAttr(Type type, const APInt &value) {
 /// a null attribute for zero width results.
 static IntegerAttr getIntZerosAttr(Type type) {
```

Comment:
Plz update the comment - this never returns null now.  Also please check for any null checks of its result that can be removed.

Reply:
Ah good catch, that one slipped through. All the uses are updated :+1: Thanks!